modules/installation-configuration-parameters.adoc

```diff
@@ -927,10 +927,14 @@ Additional Azure configuration parameters are described in the following table:
 |====
 |Parameter|Description|Values
 
-|`controlPlane.platform.azure.osDisk.diskSizeGB`
+|`controlPlane.platform.azure.osDisk.diskSizeGB`, `compute.platform.azure.osDisk.diskSizeGB`, `platform.azure.osDisk.diskSizeGB`
```

Comment:
Before combining all 3 into one cell, I'd see if we do this at all anywhere else. I don't think I've seen that done - so it's possible it might be better to add them in as their own individual rows, even if the descriptions/values are the same.

Reply:
Gotcha! Yeah I'll definitely check that.